generator/src/main/java/com/scottlogic/deg/generator/generation/fieldvaluesources/TemporalFieldValueSource.java

```diff
@@ -15,6 +15,9 @@
 
 public class TemporalFieldValueSource implements FieldValueSource {
 
+    public static final LocalDateTime ISO_MAX_DATE = LocalDateTime.of(9999, 12, 31, 23, 59, 59, 999);
+    public static final LocalDateTime ISO_MIN_DATE = LocalDateTime.of(1582, 1, 1, 0, 0);
```

Comment:
I thought we agreed a minimum date of `0001-01-01T00:00:00.000`?

Reply:
oh, sorry, i thought it was the iso minimum that doesn't require further agreement across parties?

Reply:
We documented the agreed range on #636